### ISSUE TITLE
No confidence intervals for "meanRisk"-estimates 

### ISSUE THREAD

**Issue by cqh342** [tagteam/riskRegression#20]
I am a medical student at the department of Cardiology at University Hospital of Copenhagen, Rigshopitalet, and relatively used to working in R. I am using Rstudio version: 4.0.3, riskRegression version: "2020.2.5" and OS: Windows Server 2012 R2 (version 6.3, build 9600) (provided by The Danish Institute of Statistics)

On monday the 21st of december (this week) i would get point estimates for ate-objects when assessing "meanRisk" and CI95 for both the example provided in the Vignette (Vigenette version: 08.12.2020) and when using my own data.
Today the 23rd of December - with the identical code - i get an error: "Column 'p.value' does not exist to remove".

Code example (from Vigentte version: 08.12.2020, page 12), my comments highlighted in bold text
>n <- 100
>dtS <- sampleData(n, outcome="survival")
>dtS$time <- round(dtS$time,1)
>dtS$X1 <- factor(rbinom(n, prob = c(0.3,0.4) , size = 2), labels = paste0("T",0:2))
>compute the ATE at times 5, 6, 7, and 8 using X1 as the treatment variable
>standard error computed using the influence function
>confidence intervals / p-values based on asymptotic results
>fit <- cph(formula = Surv(time,event)~ X1+X2,data=dtS,y=TRUE,x=TRUE)
>ateFit1a <- ate(fit, data = dtS, treatment = "X1", times = 5:8)
>summary(ateFit1a)
>summary(ateFit1a, short = TRUE, type = "meanRisk") **#<<< No estimates and no CI95s, error = "Column 'p.value' does not exist to remove".**
>summary(ateFit1a, short = TRUE, type = "diffRisk") **#<<< Still work including CI95s**
>summary(ateFit1a, short = TRUE, type = "ratioRisk") **#<<< Still work including CI95s**

I found an example from an older version of the Vignette (Version "2020.05.02") that, however, doesnt show CI95s for point estimates
>ateFit1b <- ate(fit, data = dtS, treatment = "X1", times = 5:8, se = FALSE) **#<<< Now i must include se = false for point estimates to work**
>summary(ateFit1a, short = TRUE, type = "meanRisk")**# <<< Estimates work, however, no CI95s**

The same errors occurs when using my own data - for purpuse of reproducability i have used examples from the vignette. 

In addition, using the band = TRUE function in the ate()-call gives the same error: "Column 'p.value' does not exist to remove". Or if trying to add bands=1 in the summary call.

I am pretty sure, from colleagues that have used this package, that it is possible to get CI95s for the risk-estimates - not just the risk-differences or the risk-ratios. Have something happened and/or do i need to do something else to get CI95s for my risk-estimates now?

Regards,
Andreas Dalsgaard Jensen

**Comment by bozenne** [comment]
Dear Andreas Dalsgaard Jensen, 

Thanks for the example. We just updated riskRegression on CRAN and I think that should fix the problem (version 2020.12.08).
```
library(riskRegression)
library(rms)
n <- 100
dtS <- sampleData(n, outcome="survival")
dtS$time <- round(dtS$time,1)
dtS$X1 <- factor(rbinom(n, prob = c(0.3,0.4) , size = 2), labels = paste0("T",0:2))
fit <- cph(formula = Surv(time,event)~ X1+X2,data=dtS,y=TRUE,x=TRUE)
ateFit1a <- ate(fit, data = dtS, treatment = "X1", times = 5:8)
summary(ateFit1a, short = TRUE, type = "meanRisk")
```

 time X1  risk          ci
    5 T0 0.462 [0.32;0.61]
    6 T0 0.557 [0.41;0.70]
    7 T0 0.575 [0.43;0.72]
    8 T0 0.575 [0.43;0.72]
    5 T1 0.509 [0.37;0.65]
    6 T1 0.607 [0.46;0.76]
    7 T1 0.625 [0.48;0.78]
    8 T1 0.625 [0.48;0.78]
    5 T2 0.388 [0.17;0.60]
    6 T2 0.476 [0.23;0.72]
    7 T2 0.492 [0.25;0.74]
    8 T2 0.492 [0.25;0.74]


So consider re-installing the package using install.packages

**Comment by cqh342** [comment]
Thank you very much :) Im not able to test the new version on the server provided by by The Danish Institute of Statistics until the administrator updates the available packages - although it works on my Mac with the test-data.
